en/components/slider/slider_ticks.md

```diff
@@ -0,0 +1,255 @@
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="slider-ticks-bottomtotop-labels-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+Here we are just disabling the **secondary tick labels** by setting [`secondaryTickLabels`]({environment:angularApiUrl}/classes/igxslidercomponent.html#secondaryticklabels) to **false** and rotate the **primary** ones by changing the [`tickLabelsOrientation`]({environment:angularApiUrl}/classes/igxslidercomponent.html#ticklabelsorientation) to **BottomToTop**(-90) through [`TickLabelsOrientation`]({environment:angularApiUrl}/enums/ticklabelsorientation.html#range) enumeration.
+
+#### Disable primary ticks and change their orientation
+Let’s move on and change the orientation of the **ticks** itself and see how we can have only **secondary ticks** enabled without any **tick labels**.
+
+```html
+<button igxButton="fab" igxRipple="white" (click)="decrease()">
+    <igx-icon fontSet="material">keyboard_arrow_left</igx-icon>
+</button>
+<div class="slider-container">
+    <igx-slider
+        [maxValue]="20"
+        [showTicks]="true"
+        [secondaryTicks]="21"
+        [secondaryTickLabels]="false"
+        [ticksOrientation]="ticksOrientation"></igx-slider>
+</div>
+<button igxButton="fab" igxRipple="white" (click)="increase()">
+    <igx-icon fontSet="material">keyboard_arrow_right</igx-icon>
+</button>
+
+```
+The two buttons you see could be used to control/update slider's **value**, but let's focus on the **ticks** manipulation. 
+
+```typescript
+  public ticksOrientation = TicksOrientation.Mirror;
+```
+
+<div class="sample-container loading" style="height: 140px">
+    <iframe id="slider-secondary-ticks-mirror-iframe" data-src='{environment:demosBaseUrl}/interactions/slider-secondary-ticks-mirror' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="slider-secondary-ticks-mirror-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+In order to change the orientation, we have used [`ticksOrientation`]({environment:angularApiUrl}/classes/igxslidercomponent.html#ticksorientation) input, which we changed from **Bottom**(default) to **Mirror**. This mirrors the visualization of the **ticks** and duplicates them at the top as well. 
+
+#### Top ticks with visible labels
+There is an edge case where our **thumb label** is hidden intentionally, and it is when [`ticksOrientaion`]({environment:angularApiUrl}/classes/igxslidercomponent.html#ticksorientation) is set to **Top** or **Mirror** and there are any visible **tick labels**. In this way, we prevent a bad user experience and overlap between the two labels. To gain a better view over that scenario let’s see the example below.
```

Comment:
maybe you mean "there are NO visible tick labels" or "there arent any visible" ?